config/smart.yml

```diff
@@ -4,9 +4,16 @@ api:
 
 auth:
   strategy: authorization_code
-  steps: []
   client_id: test-suite
   # client_secret intentionally included
   client_secret: demo-secret-s4s
   scope: launch/patient patient/*.read offline_access
   confidential_client: True
+  steps:
+    - element: '#consent-carousel'
+      execute_script: |
+        $('#consent-carousel')
```

Comment:
Note that this capability, combined with the anticipated ability to paste config snippets into our demo harness, mean that arbitrary JS invocation (within the selenium browser environment) is possible. It's okay, just something I want to be conscious of :-)